### DISCUSSION TITLE
Is there any way of changing the name of the taskwarrior command?

### DISCUSSION THREAD

**Discussion by vctls** [kdheepak/taskwarrior-tui#610] [Q&A] [answered]
I already have a command named `task`, so I had to alias taskwarrior to something else.  
Is there any way of changing the default command name used by `taskwarrior-tui`?

**Comment by RedEtherbloom** [comment, marked as answer]
Not yet, although it is planned. 
You may get away with writing a shell script that aliases task to taskwarrior and then starts taskwarrior-tui in this environment until then.

**Reply by vctls** [reply]
I tried using aliases and functions in a bash script, but it didn't work.  
In the end, I had to resort to this ugly thing:  
```bash
#!/bin/bash
cat <<EOT > ~/.local/share/taskwarrior-tui/task
#!/bin/bash
exec taskwarrior "\$@"
EOT
chmod +x ~/.local/share/taskwarrior-tui/task
export PATH=~/.local/share/taskwarrior-tui:$PATH
taskwarrior-tui
```

Edit: actually, I think I'm going to put everything into a container.  
Because of course, I have the same problem with `bugwarrior`.
